src/components/MapView/Analyser/index.tsx

```diff
@@ -13,6 +13,7 @@ import {
   FormControlLabel,
```

Comment:
This file is clearly getting out hand, let's add a TODO to split it into different files per type of analysis